amarna/command_line.py

```diff
@@ -149,9 +150,12 @@ def main() -> int:
 
     filename = args.filename
 
-    if not os.path.isabs(filename):
+    if filename is not None and not os.path.isabs(filename):
```

Comment:
We should separate the None check from the isabs check: otherwise, if the filename is None, we will not catch it until the `os.path.exists` function call below has a runtime error.